index.html

```diff
@@ -0,0 +1,611 @@
+                  <img src="/img/orange-star.svg" alt="positive star rating" title="Positive Star Rating">
+                  <img src="/img/orange-star.svg" alt="positive star rating" title="Positive Star Rating">
+                  <img src="/img/grey-star.svg" alt="negative star rating" title="Negative Star Rating">
+                  <p><span>$5.00</span> Free!</p>
+                </figcaption>
+              </figure>
+            </li>
+          </ul>
+          <div class="swiper-pagination swiper-pagination-2"></div>
+        </div>
+      </section>
+      <section data-section="download-app" class="l-flex-container download-left">
+        <img src="/img/phones.png" alt="Image of iPhones showing AppBox logo" title="AppBox iPhone" width="439" height="605">
+        <div class="download-text">
+          <h1><strong>Download</strong> Our App</h1>
+          <p class="download-text">Choose your native platform and download the app <em>free!</em></p>
+          <ul>
+            <li><a class="btn-large" target="_blank" href="https://play.google.com/store/apps?hl=en"><span class="svg-google"></span>Google Play</a></li>
+            <li><a class="btn-large" target="_blank" href="https://itunes.apple.com/us/genre/ios/id36?mt=8&cid=wwa-us-kwg-features"><span class="icon-apple2"></span>iOS App Store</a></li>
+          </ul>
+          <aside class="newsletter">
+            <form>
+              <label for="subscribe">Subscribe To Newsletter</label>
+              <input id="subscribe" type="email" name="email" placeholder="Enter your email address">
+              <button class="btn-small" type="submit">Submit</button>
+            </form>
+          </aside>
+         </div>
+      </section>
+      <section data-section="discounted-pricing" id="pricing">
+        <h1 class="header-underline">Discounted Pricing</h1>
+        <p>Nullam id odio quis risus placerat aliquam. Cum sociis natoque penatibus et magnis dis part urient montes nascetur ridiculus mus curabitur tellus lacus.</p>
+        <table class="bundle">
+          <thead>
+            <tr>
+              <th>Standard Package</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td class="bundle-plan"><span>$11</span><br> per month
+              </td>
+            </tr>
+            <tr>
+              <td class="bundle-list">Unlimited support</td>
+            </tr>
+            <tr>
+              <td>Easy &amp; Fast Customization
+            </tr>
+            <tr>
+              <td>Unlimited Color possible</td>
+            </tr>
+            <tr>
+              <td>Free PSD Template</td>
+            </tr>
+            <tr>
+              <td>Lifetime Update</td>
+            </tr>
+          </tbody>
+          <tfoot>
+            <tr>
+              <td><a class="btn-small" href="#">Buy now</a></td>
+            </tr>
+          </tfoot>
+        </table>
```

Comment:
I have no problem with using `<table>` to lay this out; one could argue that it's a "list" and should therefore be marked up as a `<ul>`, but I think `<table>` is perfectly appropriate here.

The one thing I will note is that, while I applaud your use of the full table semantics, they're actually unnecessary in this case. Since there's only one column, and you aren't using `<thead>`, `<tbody>` or `<tfoot>` for styling, those elements could have been omitted—they aren't required. Semantically, in this use case, `<th>` performs the same role as `<thead>`, without the additional row markup.